buildscripts/incremental/setup_conda_environment.sh

```diff
@@ -30,7 +30,7 @@ conda list
 # Use conda-forge for NumPy 1.24 - at the time of writing it is not available
 # on the defaults channel.
 
-if [ "${NUMPY}" == "1.24" ]; then
+if [ "${NUMPY}" == "1.24" -o "${NUMPY}" == "1.25" ]; then
```

Comment:
I think conda-forge is only needed for 1.25 these days (1.24 is in the default channels).

Reply:
This raises a bit of a concern because we've been using conda-forge packages until now for 1.24. Should we switch this off for 1.25 once Anaconda packages are ready?